daemon/oci_linux.go

```diff
@@ -463,11 +463,9 @@ func setMounts(daemon *Daemon, s *specs.Spec, c *container.Container, mounts []c
 		}
 
 		if m.Source == "tmpfs" {
-			opt := []string{"noexec", "nosuid", "nodev", volume.DefaultPropagationMode}
+			opt := []string{"noexec", "nosuid", "nodev", volume.DefaultPropagationMode, "size=65536k"}
```

Comment:
Could you add a test for specifying a custom propagation mode as well?
Also wondering how deterministic the behavior of having duplicate keys is.